Formula/allegro.rb

```diff
@@ -33,6 +34,8 @@ class Allegro < Formula
 
   on_linux do
     depends_on "gcc"
+    depends_on "jpeg-turbo"
+    depends_on "libpng"
```

Comment:
https://github.com/liballeg/allegro5/#library-dependencies

`libpng` and `jpeg`/`jpeg-turbo` are already in dependency tree. These both are directly used on Linux, but not on macOS.